src/Components/Autocomplete.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  list: Person[],
+  onChange: (value: string) => void,
+  value: string,
+  onDelayApply: (value: string) => void,
+  onSelect: (value: string) => void,
+  query: string,
+};
+
+export const Autocomplete: React.FC<Props> = React.memo(
+  ({
+    list,
+    onChange,
+    value,
+    onDelayApply,
+    onSelect,
+    query,
+  }) => {
+    return (
+      <div className="dropdown is-active">
+        <div className="dropdown-trigger">
+          <input
+            type="text"
+            placeholder="Enter a part of the name"
+            className="input"
+            onChange={(event) => {
+              onChange(event.target.value);
+              onDelayApply(event.target.value);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)